### PR TITLE
Configured the CSS sheet and made a sidebar for the home page

### PR DESCRIPTION
**Pull request by LukeM12**
This is a nicer home page for the UI. I will make this sidebar with the css configured for each page for easier and more user friendly app page navigation

### CODE REVIEW

File: app/views/users/index.html.erb [hunk 1]
```diff
@@ -30,6 +30,38 @@
   </tbody>
 </table>
 
+
```
Comment (strangestar) [inline app/views/users/index.html.erb]: Is this some example to test out the visuals?
Did you intend to commit this?

Comment (LukeM12) [inline app/views/users/index.html.erb]: Is this some example to test out the visuals?
[Yes] Table is ugly 
Did you intend to commit this?
[No]

Comment (LukeM12) [inline app/views/users/index.html.erb]: Given our timeline, might be better if I repush an updated project and re-pull request

**Comment by fadyibrahim** [comment]
Well done Luke.

**Pull request merged**